### PR TITLE
Add Openshift virtualization tests as a step

### PR DESCRIPTION
**Pull request by rabin-io**

### CODE REVIEW

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.16-cnv-odf-ocp4.16-lp-interop-cnv-odf-tests-aws-upi-ocp416

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.16-cnv-odf-ocp4.16-lp-interop-cnv-odf-tests-aws-upi-ocp416

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.16-cnv-odf-ocp4.16-lp-interop-cnv-odf-tests-aws-upi-ocp416

**Comment by rabin-io** [comment]
/pj-rehearse periodic-ci-RedHatQE-interop-testing-cnv-4.16-cnv-odf-ocp4.16-lp-interop-cnv-odf-tests-aws-upi-ocp416

File: ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+unset KUBERNETES_PORT_443_TCP_PROTO
+unset KUBERNETES_PORT_443_TCP_ADDR
+unset KUBERNETES_SERVICE_HOST
+unset KUBERNETES_PORT
+unset KUBERNETES_PORT_443_TCP_PORT
+
+set -x
+START_TIME=$(date "+%s")
+
+poetry run pytest --junitxml "${ARTIFACT_DIR}/xunit_results.xml" --pytest-log-file="${ARTIFACT_DIR}/tests.log" -o cache_dir=/tmp --tc=hco_subscription:kubevirt-hyperconverged
```
Comment (dbasunag) [inline ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh:31]: Are we creating subscription named `kubevirt-hyperconverged`?

Comment (rabin-io) [reply ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh:31]: Yes, 

https://github.com/openshift/release/blob/master/ci-operator/step-registry/interop-tests/deploy-cnv/interop-tests-deploy-cnv-commands.sh#L39

**Review by lukas-bednar — APPROVED** [review]
lgtm

File: ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh (hunk 1 above)
Comment (dbasunag) [reply ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh:31]: @rabin-io that does not seem right ```kind: Subscription
metadata:
  name: hco-operatorhub
  namespace: $CNV_INSTALL_NAMESPACE```. The name seems to be `hco-operatorhub` based on the link you shared. In that case the command you used to override hco_subscription `--tc=hco_subscription:kubevirt-hyperconverged` is not necessary, as we already have it correct in https://github.com/RedHatQE/openshift-virtualization-tests/blob/main/tests/global_config.py#L6

Comment (rabin-io) [reply ci-operator/step-registry/interop-tests/openshift-virtualization-tests/interop-tests-openshift-virtualization-tests-commands.sh:31]: I will check this once I refactor the jobs to use our cluster profile again.

**Pull request merged**